weaviate/rbac/roles.py

```diff
@@ -191,7 +191,7 @@ async def by_user(self, user: str) -> Dict[str, Role]:
             for role in await self._get_roles_of_user(user)
         }
 
-    async def users(self, user_name: str) -> Dict[str, User]:
+    async def assigned_users(self, user_name: str) -> Dict[str, User]:
```

Comment:
shouldn't we also use role_name are argument here?

Reply:
Definitely, that must've been left in from ages ago 🤔